client/src/Components/Dashboard/Dashboard.js

```diff
@@ -38,7 +40,8 @@ function Dashboard() {
         <SubmitPost />
         <PostCard
           data={data}
```

Comment:
we could read forum data using context inside postcard. so we dont need to pass data as a prop.